src/Security/ResourceAccessChecker.php

```diff
@@ -11,15 +11,6 @@
 
 declare(strict_types=1);
 
-/*
- * This file is part of the API Platform project.
- *
- * (c) Kévin Dunglas <dunglas@gmail.com>
- *
```

Comment:
move this message over `declare(strict_types=1);`

Reply:
Same.